include/seqan3/search/fm_index/bi_fm_index_cursor.hpp

```diff
@@ -1022,6 +1022,28 @@ class bi_fm_index_cursor
                    return locate_result_value_type{sequence_rank-1, sequence_position};
                });
     }
+
+    /*!\cond DEV
+     * \brief Serialisation support function.
+     * \tparam archive_t Type of `archive`; must satisfy seqan3::cereal_archive.
```

Comment:
```suggestion
     * \tparam archive_t Type of `archive`; must model seqan3::cereal_archive.
```
:nail_care:

Reply:
We use `satifsy` in every serialisation method (~12 times).  Can I do this as separated PR?

Reply:
sure